web/src/providers/TestOutput/TestOutput.provider.tsx

```diff
@@ -101,12 +105,16 @@ const TestOutputProvider = ({children, testId, runId}: IProps) => {
 
   const onSave = useCallback(
     async (outputs: TTestOutput[]) => {
-      await setTestOutputs({testId, testOutputs: toRawTestOutputs(outputs)}).unwrap();
-      const run = await reRunTest({testId, runId}).unwrap();
+      const plugin = TriggerTypeToPlugin[test?.trigger?.type || TriggerTypes.http];
```

Comment:
Should we encapsulate this into a single layer so we don't have to keep adding it for the different sections?